dissect/target/helpers/record_modifier.py

```diff
@@ -94,6 +94,13 @@ def modify_record(target: Target, record: Record, modifier_function: ModifierFun
     for field_name, resolved_path in _resolve_path_types(target, record):
         try:
             _record = modifier_function(field_name, resolved_path)
+
+        except FileNotFoundError as e:
+            target.log.info(
+                "Unable to modify record '%s' with function '%s': %s", record._desc.name, modifier_function.__name__, e
+            )
+            target.log.debug("", exc_info=e)
+
         except FilesystemError as e:
             target.log.warning(
```

Comment:
```suggestion
        except FilesystemError as e:
            level = logging.INFO if isinstance(e, FileNotFoundError) else logging.WARNING
            target.log.log(
                level,
```
Maybe a bit more concise. I played around with this or setting `target.log.warning`/`target.log.info` to a variable but I'm not sure if I have a specific preference.